src/common/transformations/src/transformations/common_optimizations/sdpa_fusion.cpp

```diff
@@ -467,148 +497,49 @@ SDPAFusionMatcherSinks::SDPAFusionMatcherSinks() {
     auto softmax_pred = consumers_count(1) && axis_predicate;
     auto softmax = wrap_type<v8::Softmax>({sinks_sub}, softmax_pred);
 
-    auto sinks_ss = wrap_type<v1::StridedSlice>({softmax, any_input(), any_input(), any_input()});
-    auto qkv = wrap_type<v0::MatMul>({sinks_ss, v});
+    auto sinks_slice = wrap_type<v1::StridedSlice>({softmax, any_input(), any_input(), any_input()}) |
+                       wrap_type<v8::Slice>({softmax, any_input(), any_input(), any_input(), any_input()});
```

Comment:
This line updates the pattern to match not only ``StridedSlice`` but also ``Slice`` op. Would be good to update the transformation test to cover the required `Slice` case as well.